components/brave_wallet/browser/zcash/zcash_shield_sync_service.cc

```diff
@@ -207,6 +207,11 @@ void ZCashShieldSyncService::OnAccountInit(
 
 void ZCashShieldSyncService::VerifyChainState(
     ZCashOrchardStorage::AccountMeta account_meta) {
+  if (!account_meta.latest_scanned_block_id) {
+    latest_scanned_block_ = account_meta.account_birthday - 1;
```

Comment:
can this overflow?

Reply:
Added additional check, block height should not be less than 1,687,104.